spec/controllers/api/did_not_votes_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe API::DidNotVotesController do
+
+  let(:user)                  { create :user }
+  let(:group)                 { create :group }
+  let(:discussion)            { create :discussion, group: group }
+  let(:motion)                { create :motion, discussion: discussion }
+
+  before do
+    motion
+    stub_request(:post, "http://localhost:9292/faye").to_return(status: 200)
+    group.admins << user
+    sign_in user
+    MotionService.close(motion)
+  end
+
+  describe 'index' do
+
+    context 'success' do
+      it 'returns did not votes for a motion' do
+        get :index, motion_id: motion.id, format: :json
+        json = JSON.parse(response.body)
+        expect(json.keys).to include *(%w[did_not_votes])
```

Comment:
Yep, I think that's what I'm getting at. Our index test for discussions looks like:

```
      it 'returns discussions filtered by group' do
        get :index, group_id: group.id, format: :json
        json = JSON.parse(response.body)
        expect(json.keys).to include *(%w[discussions])
        discussions = json['discussions'].map { |v| v['id'] }
        expect(discussions).to include discussion.id # <-- this line
        expect(discussions).to_not include another_discussion.id # <-- and this one
      end
```

Which confirms that you're a) getting discussions from the specified group and b) not getting discussions from another group leaking in.